### DISCUSSION TITLE
It's easy for you, it's very difficult for me, please help.

### DISCUSSION THREAD

**Discussion by bysmoke** [google/model-viewer#4698] [Q&A]
I want to let customers experience products such as sofas and furniture with model viewer. I don't know much about this subject. How can I ensure that products such as sofas and chairs are automatically displayed close to scale? Is there anyone who has time to help me? Thank you in advance.

**Comment by danilokrulj** [comment]
You add this parameter to your <model-viewer> element:

ar-scale="fixed" 

With that parameter the models scale will be fixed to 100%, so you just make sure the models are in the correct dimensions.

**Reply by bysmoke** [reply]
i have no coding knowledge at all. can you give me a ready-made template code that will be able to work for ios web and android, which will allow my customers to see the products with the Deceleration option. I'm experimenting, but I'm not succeeding. src="/assets/models/your_model.usdz"src="../../assets/ShopifyModels/Chair.i will be able to modify and use the glb" files. thank you for your help.

**Reply by danilokrulj** [reply]
I'm not quite sure what Deceleration option is. You can use the model-viewer editor to get a basic html page. And to add the fixed scale so that the model is always at 100% scale you just add the line i wrote to you inside the model viewer element, like this:

<model-viewer id="someId" src="pathToYourModel" ar-scale="fixed" ar ar-modes.... >

So you just add the ar-scale="fixed" inside the <model-viewer> tag.

**Reply by danilokrulj** [reply]
But if you really have no coding knowledge you might want to find someone who would help you implement what you have in mind. But if you are up for a challenge I'm sure you could figure it out if you really gave it a try. Good luck! :)

**Reply by bysmoke** [reply]
<!DOCTYPE html>
<html lang="en">
  
  <script
    type="module"
    src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.4.0/model-viewer.min.js"
  ></script>
  <model-viewer
    src="https://cdn.glitch.global/65f4191b-7948-420a-a200-4859fc5b8fdb/puff.glb?v=1709904708148"
    ar
    ar-modes="scene-viewer webxr quick-look"
    camera-controls
    tone-mapping="commerce"
    poster="poster.webp"
    shadow-intensity="1"
    camera-orbit="-71.45deg 58.82deg 1.984m"
    field-of-view="30deg"
  >
    <div class="progress-bar hide" slot="progress-bar">
      <div class="update-bar"></div>
    </div>
    <button slot="ar-button" id="ar-button">View in your space</button>
    <div id="ar-prompt">
      <img src="https://modelviewer.dev/shared-assets/icons/hand.png" />
    </div>
  </model-viewer>
</html>


![pufandroid](https://github.com/google/model-viewer/assets/162462298/a0b5096f-bb24-4613-897d-e969d9df4209)
![puffweb](https://github.com/google/model-viewer/assets/162462298/f63be6a8-d46c-4b93-b7e3-3299e57b0661)
I don't understand where is the mistake in this code? The AR button does not appear.

**Reply by elalish** [reply]
If you want us to tell you the mistake in your code, you'll need to show us your code. You'll also need to be specific about what device and browser you're using - the AR button is mobile-only and doesn't work on all browsers.
